Importer.php

```diff
@@ -406,7 +417,13 @@ public function import($idSite, $viewId, Date $start, Date $end, Lock $lock, $se
             $this->importStatus->finishImportIfNothingLeft($idSite);
 
             unset($recordImporters);
-        } catch (DailyRateLimitReached $ex) {
+
+        } catch (DailyRateLimitReached  | CloudApiQuotaExceeded $ex) {
+            if($ex instanceof CloudApiQuotaExceeded){
```

Comment:
Just FYI, the error message shows the limit is 5k, even though its much less and can give wrong indication
![Screenshot from 2022-11-21 13-00-10](https://user-images.githubusercontent.com/14902277/202991828-a6d96b08-44b7-4ae7-983e-89a173000f02.png)

Reply:
Hi @AltamashShaikh, the error message given in the [CloudApiQuotaExceeded](https://github.com/matomo-org/plugin-GoogleAnalyticsImporter/pull/295/files/ca3362ad9cc1c7a5c5fd3e624cf2d43bdf0ce5ea#diff-0efeffe2a11b1e4d97a48ccc2eacaba282b3b3031dff71cb507ca9e4a9e7233b) exception has a different text. Looks like you are getting the "DailyRateLimitReached" which was already there.
Just wondering if you actually got rate limited from Google.

Reply:
@elabuwa I did get a CloudException error

Reply:
Update added @AltamashShaikh